lib/matplotlib/dates.py

```diff
@@ -96,11 +96,11 @@
       <../gallery/ticks_and_spines/date_demo_rrule.html>`_.
 
     * :class:`AutoDateLocator`: On autoscale, this class picks the best
-      :class:`RRuleLocator` to set the view limits and the tick
+      :class:`DateLocator` to set the view limits and the tick
```

Comment:
I think in general its more useful to point to the `RRuleLocator` than the generic `DateLocator`.  If you want to be pedantic, you can refer to both.

Reply:
In my first draft I had been referring to both, but then found it too complicated/pedantic for this description. The ``Yearlocator`` is also no ``RRuleLocator``.